server/@types/documents.d.ts

```diff
@@ -13,6 +13,7 @@ export interface DocumentCategoryMetadata {
   multiple?: boolean
   versioned?: boolean
   fileNamePatterns?: string[]
+  showOnFullView?: boolean
```

Comment:
I think you said this was redundant?

Reply:
Hmm if so, I changed :) I think it is good as it is a check of the html representation. I had a bug in this part though that we fixed. Do you think I should delete it?

Reply:
are you using it anywhere?